packages/user-profile-spa/src/app/user-profile-components/user-details/user-details.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { UserDetailsMock } from '../../mocks/user-profile-mock';
+import { AppService } from 'src/app/app.service';
+@Component( {
+  selector: 'app-user-details',
+  templateUrl: './user-details.component.html',
+  styleUrls: [ './user-details.component.scss' ],
+} )
+export class UserDetailsComponent implements OnInit
+{
+  userDetails: any;
+  showDetails = true;
+
+  constructor( private appService: AppService ) { }
+
+  ngOnInit(): void
+  {
+    this.userDetails = UserDetailsMock;
```

Comment:
It should call the User Service API to get the full details of the user.\
SSO won't give you all the details, just some basic stuff.

Reply:
This is actually work in progress. It would just take the rhuuid from the SSO and rest would be from user service